docs/data/update-oasis.md

```diff
@@ -3,11 +3,11 @@ Updating Software in OASIS
 
 OASIS is the OSG Application Software Installation Service. It is the recommended method to install software on the Open Science Grid. It is implemented using CernVM FileSystem (CVMFS) technology.
 
-This document is a step by step explanation of how a Virtual Organization (VO) Software Adminstrator can enable the OASIS service and use it to publish and update software on OSG Worker Nodes under `/cvmfs/oasis.opensciencegrid.org`.
+This document is a step by step explanation of how a Virtual Organization (VO) Software Administrator can enable use of the shared OASIS service and use it to publish and update software on OSG Worker Nodes under `/cvmfs/oasis.opensciencegrid.org`.
+The shared OASIS service is especially appropropriate for VOs that have a relatively small number of members and a relatively small amount of software to distribute.  Larger VOs should consider [hosting their own separate repositories](external-oasis-repos).
 
 !!! note
-    -   For information on how to configure a client for OASIS see the [CVMFS installation documentation](../worker-node/install-cvmfs).
-    -   For information on hosting your own repository see the [OASIS repository installation](external-oasis-repos).
+    For information on how to configure a client for OASIS see the [CVMFS installation documentation](../worker-node/install-cvmfs).
```

Comment:
```suggestion
    For information on how to configure a client for OASIS see the [CVMFS installation documentation](/worker-node/install-cvmfs).
```

Minor suggestion to use [site-relative links](https://opensciencegrid.org/technology/documentation/style-guide/#links)

Reply:
But there are a whole bunch of similar links in this file and directory, and I didn't even check the rest of the documentation.  I think if you're going to change this it should be with a separate PR that changes everything.

Reply:
That's reasonable